### PR TITLE
Message v2

### PR DESCRIPTION
**Pull request by avbel**
Make sure you've checked off all these things before submitting:

- [ ] This pull request contains 100% test coverage.
- [ ] This pull request is completely documented. All available fields are listed in the docs with description.
- [ ] Each public facing function should include at least one @example in the jsdoc.
- [ ] Run `npm test` before committing.

### CODE REVIEW

**Comment by dtolb** [comment]
Can we take out yarn.lock?

File: test/message-v2-test.js [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+		});
+	});
+	describe("createApplication()", function () {
+		before(function () {
+			nock.disableNetConnect();
+			nock("https://dashboard.bandwidth.com")
+				.persist()
+				.post("/v1.0/api/accounts/id/applications",
+					"<Application><AppName>App1</AppName><CallbackUrl>url</CallbackUrl><CallBackCreds/></Application>")
+				.reply(200, "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>" +
```
Comment (dtolb) [inline test/message-v2-test.js]: better to keep these XML responses requests in another file and use template strings? Could help update tests if these payloads ever need to get updated

Comment (avbel) [reply test/message-v2-test.js]: Done. Moved to `templates.json`

**Comment by avbel** [comment]
About lock file. Lets use yarn anywhere. I removed npm lock file

**Pull request merged**